exercises/ex1.cpp

```diff
@@ -1,48 +1,36 @@
-/* Improve this program using a switch-case. */
-
 #include <iostream>
 using namespace std;
 
-int main()
-{
-  int week;
 
-  /* Input week number from user */
-  cout << "Enter week number(1-7): " << endl;
-  cin >> week;
+int main() {
 
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
+    int week;
+    cout << "Enter week number(1-7): " << endl;
+    cin >> week;
 
-  return 0;
-}
\ No newline at end of file
+    switch (week){
+        case 1:
+            cout << "Monday" << endl;
+            break;
+        case 2:
+            cout << "Monday" << endl;
+            break;
+        case 3:
+            cout << "Wednesday" << endl;
+            break;
+        case 4:
+            cout << "Thursday" << endl;
+            break;
+        case 5:
+            cout << "Friday" << endl;
+            break;
+        case 6:
+            cout << "Saturday" << endl;
+            break;
+        case 7:
+            cout << "Sunday" << endl;
+            break;
+        default:
+            cout << "Invalid input! Please enter week number between 1-7." << endl;
+    }
```

Comment:
manca return 0, inoltre, potresti mettere un `cout << endl;` dopo lo switch e toglierlo in tutti i case in modo da ridurre la ridondanza nel codice